open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/registration/EngineServiceDescription.java

```diff
@@ -251,9 +290,9 @@ public String getEngineServicePartnerService()
 
 
     /**
-     * Return the type of governance engine that this engine service supports.
+     * Return the open metadata type of governance engine that this engine service supports.
```

Comment:
should there be a `the` before governance engine?

Reply:
Please feel free to change it